bin/p2-exec/main.go

```diff
@@ -80,20 +81,55 @@ func main() {
 		}
 	}
 
+	cgroupPath := ""
+	if *podID != "" {
+		cgConfig, err := cgGetConfig("RESOURCE_LIMITS", "")
+		if err != nil {
+			log.Fatal(err)
```

Comment:
we should consider whether we want this to actually be an error. I suspect that we've trained other people to use p2-exec in contexts where this env var won't exist. Maybe the preparer should write a `--enforce-resource-limit` flag that turns on this strict code?

Reply:
actually maybe the fact that pod ID is provided already covers what i'm saying. Nobody would pass the pod ID in a one-off invocation